README.md

```diff
@@ -43,11 +43,15 @@ fail during the verification phase if you don't follow them.
 
     mvn clean package
     
-Run findbugs for future reference
-or to make sure you haven't introduced any new warnings
+Run findbugs for future reference or to make sure you haven't introduced any
+new warnings
 
     mvn findbugs:findbugs
-    
+
+Run checkstyle
+
+    mvn checkstyle:checkstyle
```

Comment:
Right, but running it explicitly means you get the log output without all the noise of the compile and test output.